### DISCUSSION TITLE
Welcome to cp-wiki Discussions!

### DISCUSSION THREAD

**Discussion by lucifer1004** [lucifer1004/cp-wiki#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by j0k4rX** [comment]
I saw your code of atcoder. And I am so glad that you're using rust as a means of solving competitive problems. 
I am also interested in rust.
Are u on discord so that I can contact you?

**Reply by lucifer1004** [reply]
I think GitHub Discussions is an ideal place for such discussion.

I started using Rust a few months ago, and I am still far from proficient. I think the best way to learn a language is to start using it as often as you can. At first, it might seem hard. But things would become smoother as you move on.
